addons/survey/static/src/js/survey_form.js

```diff
@@ -1109,6 +1112,16 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
                 $(dependingQuestion).find('textarea').val('');
             }
         }
+        // Sections visibility
+        if (this.options.questionsLayout === 'one_page'){
```

Comment:
Linting: should be a space between the ending parenthesis and the opening bracket.